klippa/src/glyf_loca.rs

```diff
@@ -121,12 +184,39 @@ fn subset_simple_glyph(g: &SimpleGlyph) -> Vec<u8> {
     if i == 0 {
         return out;
     }
-    let trimmed_len =
-        10 + 2 * (g.number_of_contours() as usize) + 2 + g.instruction_length() as usize + i;
-    let Some(trimmed_slice) = g.offset_data().as_bytes().get(0..trimmed_len) else {
+
+    let glyph_bytes = g.offset_data().as_bytes();
+    let header_len = 10 + 2 * (g.number_of_contours() as usize) + 2;
+    let Some(header_slice) = glyph_bytes.get(0..header_len) else {
         return out;
     };
+    out.extend_from_slice(header_slice);
+
+    if plan
+        .subset_flags
+        .contains(SubsetFlags::SUBSET_FLAGS_NO_HINTING)
+    {
+        out[header_len - 2] = 0;
+        out[header_len - 1] = 0;
```

Comment:
I think this would benefit from a comment like, 'set instructionLength field to 0'